battleship.py

```diff
@@ -247,19 +206,19 @@ def test_shot(selection,board): #checks if shot is valid
 	return test
 
 def write_ship(selection,board): #writes the (now checked) ship selection to the board
-	writecell = 0
+	# writecell = 0
```

Comment:
This wasn't being used, but I wasn't sure if it had a future use, so I commented it out.